controllers/state_of_the_world.go

```diff
@@ -294,13 +296,28 @@ func (b *BootOptionsBuilder) getConsolePluginOptions() []controller.ControllerOp
 	return opts
 }
 
+func (b *BootOptionsBuilder) getDNSOperatorOptions() []controller.ControllerOption {
+	var opts []controller.ControllerOption
+	opts = append(opts,
+		controller.WithRunnable("dnsrecord watcher", controller.Watch(&kuadrantdnsv1alpha1.DNSRecord{}, DNSRecordResource, metav1.NamespaceAll)),
```

Comment:
I think this needs address as it brings in results that are not created by kuadrant and kuadrant has no right to know about.

Reply:
Yes, so i actually wasn't aware until you commented and i looked more closely at the controller code that there was a way to filter out the objects that get added to the topology. This is definitely something i think we should do as we don't want to interfere with resources that weren't created by a policy as you correctly pointed out.

The only way i can see we can really do that with the current implementation of the controller is by using a label filter, and ensuring that all resources we create on behalf of the kuadrant operator are consistently labelled. I had a go at that here https://github.com/Kuadrant/kuadrant-operator/pull/937/commits/8e1afd24629047c0e2c11c6767b6eff3dbbb9d8b.

If this is the approach we want to take, i think we should do it for all resources that we create on behalf of any policy. I noticed for instance TLS Certificates and Issuers that are not created by a TLSPolicy are also added to the topoloy.

@guicassolato @KevFan @Boomatang wdyt about common labelling for all created resources?